### ISSUE TITLE
8$: warn on missing '=>' in one case

### ISSUE THREAD

**Issue by strager** [quick-lint/quick-lint-js#814]
This example should report a missing '=>', but it doesn't:
```js
        this._customComponents[name] = (attributes) {
        };
```
We already warn on missing '=>' in other cases. I don't know why this case has different warnings.

**Comment by rol1510** [comment]
I claim this for-hire task. I expect payment after I complete this task. I will email the quick-lint-js team if I am assigned this task.

**Comment by strager** [comment]
> I claim this for-hire task. I expect payment after I complete this task. I will email the quick-lint-js team if I am assigned this task.

For future reference, you should not claim two open for-hire tasks at once. You already had #741 claimed.
